tests/engine/render/effect/bloom/test.js

```diff
@@ -1,17 +1,12 @@
-if (typeof user === 'undefined') user = "highfidelity/";
-if (typeof repository === 'undefined') repository = "hifi_tests/";
-if (typeof branch === 'undefined') branch = "master/";
-
-var autoTester = Script.require("https://github.com/" + user + repository + "blob/" + branch + "tests/utils/autoTester.js?raw=true" );
+Script.include("https://raw.githubusercontent.com/highfidelity/hifi_tests/master/tests/utils/branchUtils.js");
+var autoTester = createAutoTester(Script.resolvePath("."));
 
 autoTester.perform("effect - bloom", Script.resolvePath("."), "secondary", function(testType) {
     var pos =  MyAvatar.position;
 
-    var TESTS_URL = "https://github.com/" + user + repository + "blob/" + branch;
-    var SUFFIX = "?raw=true";
-    var RAW_TESTS_URL = "https://raw.githubusercontent.com/" + user + repository + branch;
+    var TESTS_URL = "https://raw.githubusercontent.com/highfidelity/hifi_tests/" + autoTester.getBranch() + "/";
```

Comment:
like we said before, we shouldn;t recreate thee absolute path
instead express it relative to the getRepositoryPath and maybe other helper functions (getTestsPath() or getAssetsPath())